eclipse-scout-core/src/form/fields/LookupBox.less

```diff
@@ -43,6 +43,23 @@
         top: 0;
         height: 100%;
       }
+
+      .active-filter-undefined&.inactive {
+        font-style: italic;
+
+        & > .table-cell.last > .text::after {
+          content: ' (' var(--inactive-lookup-row-suffix-text) ')';
+        }
+      }
+    }
+
+    .active-filter-undefined& > .tree-data > .tree-node.inactive,
```

Comment:
Does this work in the Scout classic widgets app? In my test, the suffix is added even if only inactive elements are shown (inactive radio button checked)